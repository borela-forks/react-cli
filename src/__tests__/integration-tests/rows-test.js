// @flow strict

import React from "react";
import TestRender from "./test-render";

TestRender(
  "Rows should render on new lines",
  <div>
    1st row
    <br />
    2nd row
    <br />
    3rd row
  </div>
);
