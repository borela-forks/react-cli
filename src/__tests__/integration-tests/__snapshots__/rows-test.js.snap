// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Rows should render on new lines 1`] = `
"1st row+++++++++++++++++++++++++++++++++++++++++++
2nd row+++++++++++++++++++++++++++++++++++++++++++
3rd row+++++++++++++++++++++++++++++++++++++++++++
"
`;
